TestResultSummaryService/perf/DataManagerAggregate.js

```diff
@@ -0,0 +1,58 @@
+const BenchmarkMath = require( './BenchmarkMathCalculation' );
+const math = require('mathjs');
+const ObjectID = require( 'mongodb' ).ObjectID;
+
+class DataManagerAggregate {
+    async aggDataCollect(childBuild) {
```

Comment:
I think both functions could be `static`

Reply:
Changed